test/web-auth/web-auth.test.js

```diff
@@ -365,22 +365,6 @@ describe('auth0.WebAuth', function() {
             };
           });
         });
-        it('sets ssodata with a connection and without a sub when there is no payload', function(
```

Comment:
why this was removed?

Reply:
Not sure.. I put it back and added the code changes